tests/lib.rs

```diff
@@ -465,3 +465,21 @@ fn serialize_pathological_branching() {
 
     assert_eq!(deserialized, original);
 }
+
+#[test]
+fn remove_prefix() {
```

Comment:
```suggestion
fn issue_22_regression_remove_prefix() {
```

Just documenting why this is here.